src/fights.ts

```diff
@@ -1307,6 +1308,7 @@ const freeFightSources = [
           .abort()
           .setAutoAttack();
         mapMonster($location`The Haiku Dungeon`, $monster`amateur ninja`);
+        if (choiceFollowsFight()) runChoice(-1);
```

Comment:
Wouldn't this be simpler if moved inside of `mapMonster` in lib.ts?

Reply:
That's only because it was set as the auto attack. I believe sometimes the map monster call was used when a macro was not set as the auto attack.